.github/pull_request_template.md

```diff
@@ -1,17 +1,21 @@
 # Ticket 🎫
-
-This closes https://github.com/dot-base/medical-dashboard/issues/XXXX
-
-BREAKING CHANGE: some breaking change description or DELETE this line. 🚨
+This closes https://github.com/dot-base/dotclinic/issues/XXXX
 
 # Description 📖
 Provide a description of the changes. Also include motivation and context of the changes.
 
 ## Is there something controversial ? 🚨
-
 Take sometime time to explain your choices.
 This can also include further thoughts, ideas and improvements.
 
 # How to Test? 🧪
-
 Provide a list of instructions how to test the changes of this PR.
+
+# Breaking Changes ⚡️
+This PR has
+- [ ] no migrations
+- [ ] no API changes
+- [ ] no config changes
+- [ ] no changes that require a change in other repos
+
+BREAKING CHANGE: If you cannot tick all boxes, there is a breaking change. In this case, add a description here after the keyword. Otherwise DELETE this line.
```

Comment:
```suggestion
BREAKING CHANGE: If you cannot tick all boxes, there is a breaking change. In this case, add a description here after the keyword. Otherwise DELETE this line or it will falsely trigger a major version change.
```